showcase/src/assets/examples/breadcrumb/breadcrumb-config-input.ts

```diff
@@ -0,0 +1,51 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import { StarkBreadcrumbConfig } from "@nationalbankbelgium/stark-ui";
+const componentName: string = "StarkBreadcrumb";
```

Comment:
The `componentName` has to have the same value then the selector. Please change it for the selector value.